### DISCUSSION TITLE
And workarounds to get rid of the crash related to https://github.com/cefsharp/CefSharp/issues/4668

### DISCUSSION THREAD

**Discussion by ShengxiZhang** [cefsharp/CefSharp#4669] [General]
Hi experts, 
We need to support drag-drop behavior so we have to get this [pr](https://github.com/cefsharp/CefSharp/pull/4649) and upgrade CefSharp to version 120.1.x. But then we encountered this [crash](https://github.com/cefsharp/CefSharp/issues/4668). Do you have any other workarounds or suggestions about that? It's not easy for us to use different cache folders for difference instances as we need to remember some settings.
cc @amaitland 
Thank you!

**Comment by amaitland** [comment]
As there is already an issue let's keep the conversation in once place (#4668)
